### PR TITLE
Move guest wait

### PR DESCRIPTION
**Pull request by Tainan404**
### What does this PR do?
	
This PR moves the code of gues-wait.html from bbb-client (that was removed from packages) and add it to bbb-html using the meteor [assets](https://docs.meteor.com/api/assets.html).

### CODE REVIEW

**Comment by Tainan404** [comment]
Related with #9271.

**Comment by mrkeksi** [comment]
Thanks, i wanted to add this first. but on resume (click again in the demo). the outer points do not move anymore. https://codepen.io/MrKeksi/full/wvMPrEG
testet on safari. Chrome has no issues 

i have not tested the complete site but in safari it still showed a dot in your PR after stopping the animation.

File: bigbluebutton-html5/private/static/guest-wait/guest-wait.html [hunk 1]
```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Guest Lobby</title>
+  <meta charset="UTF-8">
+  <style>
+  :root{
+    --enable-animation: 1;
```
Comment (mrkeksi) [inline bigbluebutton-html5/private/static/guest-wait/guest-wait.html:8]: It's --enableAnimation

File: bigbluebutton-html5/imports/startup/server/index.js [hunk 2]
```diff
@@ -227,6 +228,21 @@ WebApp.connectHandlers.use('/useragent', (req, res) => {
   res.end(response);
 });
 
+WebApp.connectHandlers.use('/guestWait', (req, res) => {
+  if (!guestWaitHtml) {
+    try {
+      guestWaitHtml = Assets.getText('static/guest-wait/guest-wait.html');
+    } catch (e) {
+      Logger.warn(`'Could not process guest wait html file: ${e}`);
```
Comment (antobinary) [inline bigbluebutton-html5/imports/startup/server/index.js:236]:
```suggestion
      Logger.warn(`Could not process guest wait html file: ${e}`);
```

**Comment by antobinary** [comment]
Closes #7590

**Review by antobinary — APPROVED** [review]

**Comment by Beiri22** [comment]
This is not working in 2.3-alpha4, see https://github.com/bigbluebutton/bigbluebutton/issues/11053

**Comment by alauzon** [comment]
I think that we are in a loop here. This refers to  [#11053](https://github.com/bigbluebutton/bigbluebutton/issues/11053), that then refers to #9979, that then refers to here. So which issue is the one to look for  about resolving the waiting room problem?

**Comment by alauzon** [comment]
Like Beiri22 I tested it with  2.3-alpha4 and still having a 404 not found error when the guest clicks on join on the room.

**Comment by ratondeau** [comment]
This ist still not working for 2.3.0-alpha4 (1476)

The Settings in bigbluebutton.properties seems to be ok.
I get to page https://bbb.example,com/html5client/guestWait?sessionToken=wb2ema4aXXXXXXX. And then see the 404. 

What can I do to fix it. is there a file missing?

Regards Matt

**Comment by stagestars** [comment]
Agreed, still an issue in 2.3.0-alpha4 

PLEASE does some one have a work around?!

**Comment by ffdixon** [comment]
We'll have 2.3-alpha5 out soon with working moderator approval for guest waiting.

**Comment by human-bieng** [comment]
> 
> 
> We'll have 2.3-alpha5 out soon with working moderator approval for guest waiting.
We care waiting for it 
Thanks in advance

**Comment by ffdixon** [comment]
We've released 2.3-alpha5.  Please see [release notes](https://github.com/bigbluebutton/bigbluebutton/releases/tag/v2.3-alpha-5) and steps to [install/upgrade](https://docs.bigbluebutton.org/dev/dev23.html).

**Pull request merged**